### PR TITLE
Add common methods to get tenant qualified URLs.

### PR DESCRIPTION
**Pull request by Thisara-Welmilla**
Add common methods to get tenant qualified URLs which can be utilized to build the tenanted qualified URLs for the backend integration test.

### CODE REVIEW

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/identity/integration/common/utils/ISIntegrationTest.java [hunk 1]
```diff
@@ -143,6 +148,62 @@ public void setSystemproperties() {
...
+        Instance instance = isServer.getInstance();
+        String httpsPort = isServer.getInstance().getPorts().get(PRODUCT_GROUP_PORT_HTTPS);
+        String hostName = UrlGenerationUtil.getWorkerHost(instance);
+
+        if(httpsPort != null) {
+            baseURL = PRODUCT_GROUP_PORT_HTTPS + "://" + hostName + ":" + httpsPort;
+        } else {
+            baseURL = PRODUCT_GROUP_PORT_HTTPS + "://" + hostName;
+        }
+        return baseURL;
```
Comment (chamathns) [inline modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/identity/integration/common/utils/ISIntegrationTest.java:204]:
```suggestion
        if(httpsPort != null) {
            return PRODUCT_GROUP_PORT_HTTPS + "://" + hostName + ":" + httpsPort;
        } 
        return PRODUCT_GROUP_PORT_HTTPS + "://" + hostName;
```

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/identity/integration/common/utils/ISIntegrationTest.java [hunk 2]
```diff
@@ -143,6 +148,62 @@ public void setSystemproperties() {
...
+     * Get the qualified endpoint URL without the hostname for the given tenant.
+     *
+     * @param endpointURLWithHostname   The endpoint URL without the hostname.
+     * @param tenantDomain              Tenanted domain.
+     * @return Tenant qualified URL without hostname.
+     */
+    public String getTenantQualifiedURLWithoutHostName(String endpointURLWithHostname, String tenantDomain) {
+
+        if(!tenantDomain.equalsIgnoreCase(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME)) {
+            endpointURLWithHostname = TENANTED_URL_PATH_SPECIFIER + tenantDomain + endpointURLWithHostname;
```
Comment (chamathns) [inline modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/identity/integration/common/utils/ISIntegrationTest.java:182]: don't we need to add "/"

Comment (Thisara-Welmilla) [reply modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/identity/integration/common/utils/ISIntegrationTest.java:182]: The `TENANTED_URL_PATH_SPECIFIER` contains the "/" before the tenant domain and `endpointURLWithHostname` always starts with a "/".

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/identity/integration/common/utils/ISIntegrationTest.java (hunk 1 above)
Comment (Thisara-Welmilla) [reply modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/identity/integration/common/utils/ISIntegrationTest.java]: Addressed with commit fb8c8d4154bb869eb070374dde7ee5a4f1065ea0

**Review by chamathns — APPROVED** [review]

**Pull request merged**